.github/CODEOWNERS

```diff
@@ -5,4 +5,4 @@
 
 # These owners will be the default owners for everything in
 # the repo, unless a later match takes precedence.
-*       @wsm95 @mayank99
+*       @wsm95 @mayank99 @Guillar1 @Aidas-Vaiksnoras
```

Comment:
could you remove me? 😅 i don't really work on this anymore

Reply:
I could probably be removed as well. I haven't worked on Sync stuff in about a year so someone more plugged in should probably own this.

Reply:
👍